### PR TITLE
feat: Add healthcheck endpoint

### PR DESCRIPTION
**Pull request by ArtemFo**
feat: Add healthcheck endpoint

Part of [#14030](https://github.com/QualityUnit/la-issues/issues/14030)

### CODE REVIEW

File: generate.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-java -XX:MaxPermSize=256M -Xmx1024M -jar "../swagger-codegen/modules/swagger-codegen-cli/target/swagger-codegen-cli.jar" generate -i "../LiveAgent/asterisk-la/server/swagger/calls_internal.yaml" -l python -o "." -c ./client.json
```
Comment (mbebjak) [inline generate.sh:3]: Why this change?

Comment (ArtemFo) [reply generate.sh:3]: It is my personal paths. Were committed accidentally

**Review by ruslanlazin — APPROVED** [review]
